config/settings.yml

```diff
@@ -146,3 +146,5 @@ paperclip:
 
 price_policy_note_options: ~
 order_detail_price_change_reason_options: ~
+
+time_zone: "Central Time (US & Canada)"
```

Comment:
Not a big deal, but I would maybe move this higher up in the file. It won't get changed often, but it's moderately important to the behavior of the app.